.github/workflows/ci.yml

```diff
@@ -10,7 +10,8 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  FORC_VERSION: 0.14.4
+  ESCROW_FORC_VERSION: 0.14.5
+  MULTISIG_FORC_VERSION: 0.15.1
```

Comment:
The projects are all meant to be independent of each other. If 1 project is dormant for a while because it doesn't require any updates and then other projects get updates that contain breaking changes then using the same version will break other projects which is entirely not the point. They should all be self contained and use their own versions otherwise an update to 1 project may require updates to every other project when it is unnecessary

Reply:
No, the entire repo should be of projects that all work with the latest version of the compiler. It wouldn't be "an update to 1 project," it would be an update to the entire repo.

Reply:
Thought experiment: a month from now, what use is a contract that only compiles with `v0.14.5`? Almost nothing. What use is the same contract that compiles with the version of the compiler at that time? Significant!

Reply:
The point is to eventually containerise each application. They would be entirely independent from everything else. The version can be locked in the container until it requires an update otherwise it can remain there as it is on the older version of the compiler. There is no point in creating additional work and constantly updating something that does not need to be updated. As more applications are added, and with the rate of change of the compiler, any 1 small change will require days to weeks of work just to bump a version in 1 application that needs it. It's just wrong.

Reply:
Unless ofc you expect to maintain these as actual production (and in use) applications and not just examples that a user is meant to reference, clone and update for their own use.
If you want to keep updating the apps and redeploying the code for each update... okay. That seems wasteful but then it makes sense to keep everything up to date.